mcli/mcli-1b-eval.yaml

```diff
@@ -29,7 +29,7 @@ parameters:
       model_max_length: ${max_seq_len}
 
   model:
-    name: mosaic_gpt
+    name: mpt
```

Comment:
Should the model name be `'mptforcausallm'`

Reply:
Hmm I guess `mpt_for_causal_lm` is more correct. WDYT @dakinggg

Reply:
I think to match our current registry setup it would be `mpt_causal_lm`? since we haven't quite finished the merging to just use the same `HFCausalLM` class. https://github.com/mosaicml/llm-foundry/blob/1c8704b6c59a7fc8905560fdfa2671009253fb10/llmfoundry/model_registry.py#L8-L13

Reply:
ok I will adjust